lib/radiator_web/graphql/admin/types.ex

```diff
@@ -5,6 +5,29 @@ defmodule RadiatorWeb.GraphQL.Admin.Types do
 
   alias RadiatorWeb.GraphQL.Admin.Resolvers
 
+  @desc "The authenticated User"
+  object :user do
+    field :name, :string
+    field :email, :string
+    field :status, :string
+
+    field :display_name, :string do
+      resolve fn user, _, _ -> {:ok, user.person.display_name} end
+    end
+
+    field :nick_name, :string do
+      resolve fn user, _, _ -> {:ok, user.person.nick_name} end
+    end
+
+    field :gender, :string do
```

Comment:
Wondering if we should leave out the `gender` part, and maybe do a more generic tagging of people that the clients can choose how to fill? e.g. if they want a male/female/cis/trans/nonbinary/etc. tag. Hard baking gender doesn't seem appropriate anymore, even as a freeform text field.

Reply:
Removed gender for now and created a separate issue https://github.com/podlove/radiator/issues/126